juha/programmers/9/실패율.js

```diff
@@ -0,0 +1,16 @@
+function solution(N, stages) {
+  const answer = [];
+  const result = new Array(N + 1);
+  let total = stages.filter((stage) => stage >= 1).length;
```

Comment:
```suggestion
  let total = stages.length;
```
문제 조건을 보면 'stage 값은 1 이상 N + 1 이하의 자연수가 담겨있다.' 라고 하는데 굳이 불필요한 filter 사용인 거 같아서 그냥
stages.lenght; 만 해주면 될 거 같네요 ㅎ.ㅎ